test/SILOptimizer/devirt_conditional_conformance.swift

```diff
@@ -44,13 +44,41 @@ func genericLayer<T: Bar>(_ x: T) {
 // CHECK-LABEL: sil @$s30devirt_conditional_conformance12throughLayeryyF : $@convention(thin) () -> ()
 // CHECK: function_ref @$s30devirt_conditional_conformance10foo_markeryyF
 // CHECK: function_ref @$s30devirt_conditional_conformance10bar_markeryyF
+// CHECK: return
```

Comment:
For future reference, we usually use `CHECK-LABEL: } // end sil <name>` when the goal is to prevent matching across function bodies. It does the job better and the fact that there's a `return` instruction seems irrelevant to the test.